src/components/utils/__integrations__/onfidoApi/document.integration.js

```diff
@@ -36,7 +36,7 @@ const TEST_DOCUMENT_DATA = {
 
 describe('API uploadDocument endpoint', () => {
   beforeEach(async () => {
-    jest.setTimeout(10000)
+    jest.setTimeout(15000)
```

Comment:
Interesting that the JWT needs a bit more time when running on Github Action jobs. 🤔

Reply:
Yes, for some reason some times it fails saying that the callback didn't run within the specified timeout